lib/rules/media-feature-name-value-whitelist/README.md

```diff
@@ -0,0 +1,65 @@
+# media-feature-name-value-whitelist
+
+Specify a whitelist of allowed media feature and value pairs.
+
+```css
+@media screen and (min-width: 768px) {}
+/**                    ↑        ↑
+ *         These features and these values */
+```
+
+**Caveat:** Media feature names within range or boolean context are ignored.
+
+## Options
+
+```json
+{
+  "max-width": ["Array", "of", "values"],
+  "/regex/": ["/regex/", "non-regex"]
+}
+```
+
+If a media feature is found, only its whitelisted property values are allowed. The rule reports all
+non-matching values.
+
+If a property or value is surrounded with `/` (e.g. `"/width$/"`), it is interpreted as a regular
+expression. For example, `/width$/` will match `max-width` and `min-width`.
+
+## Examples
+
+Given
```

Comment:
"Given:"

(It's a little clunky, but the stylelint website [needs the semicolon](https://github.com/stylelint/stylelint.io/blob/master/plugins/remark-stylelint-pattern-validity/index.js#L16) for the pattern background colours).